docs/http-api.md

```diff
@@ -389,16 +352,6 @@ GET /addresses/:address/parcels
 
 Returns all the parcels that belong to a given address
 
----
-
-```
-GET /parcels/:x/:y
-```
-
-### Description
-
-Returns a single parcel by its coords
-
```

Comment:
Why are we deleting the docs for this two endpoints?
Also, correct me if I'm wrong @cazala but this is now part of the docs here: https://docs.decentraland.org/decentraland/api/ so maybe we should just leave a link here: